includes/base_controls/QControlProxy.class.php

```diff
@@ -62,7 +60,7 @@ public function RenderAsHref($strActionParameter = null, $blnDisplayOutput = tru
 			else
 				$this->strTargetControlId = $this->objForm->GenerateControlId();
 			
-			$this->strActionParameter = $strActionParameter;
+			$this->mixActionParameter = $strActionParameter;
```

Comment:
Okay. I had not checked out the code. I have been trying to setup the test environment for alpha but I am too busy with my current work. Can you please write a small guide about how to setup the test environment?

Should I just run composer and then overwrite the directory with git clone?